pkg/license/license_unit_test.go

```diff
@@ -70,6 +71,12 @@ func TestCacheData(t *testing.T) {
 }
 
 func TestFindLicenseFiles(t *testing.T) {
+	if runtime.GOOS == "windows" || runtime.GOOS == "darwin" {
+		t.Skipf("skipping test for %s", runtime.GOOS)
```

Comment:
Let's skip the test for now. But we should instead build a scenario that works on all platforms. Just dropping the `license` file (or conditionally writing it) could be a good start.

I'll put a hold on the PR so you can push a change if you want, or lift the hold and we can improve the test in a followup PR :)

Reply:
we can write that in another directory, the issue is in the same directory. WDYT?

Reply:
We can, but consider this: the idea of the test is that given several possible candidates the license classifier picks consistently the same license file. Since we cannot have LICENSE and license on case insensitive filesystems, we can skip writing the file entirely as that scenario would never happen on those.

Reply:
PTAL @puerco 
thanks for your review and feedback